lib/pavilion/plugins/results/table.py

```diff
@@ -41,13 +41,27 @@ def get_config_items(self):
                           "nested dictionaries by columns. Default False. "
                           "Only set if `has_header` is True. "
                           "Otherwise, Pavilion will ignore."
+            ),
+            yc.StrElem(
+                'title', 
+                help_text="Partial or full title of table."
+            ),
+            yc.StrElem(
+                'row_num',
+                help_text="Number of row numbers, including column names."
```

Comment:
What happens with tables with a dynamic number of rows? Do the lines skipped after the header count?

Also, "Number of rows, include the column header row."